source/server/listener_impl.cc

```diff
@@ -427,9 +526,13 @@ ListenerImpl::~ListenerImpl() {
     // parent's initManager to get ready().
     listener_init_target_.ready();
   }
+
+  // TODO(lambdai): Reorder the members to simplify the dtor.
```

Comment:
Please just do this now. It should be a trivial change.

Reply:
done